### PR TITLE
Contact form feature

### PR DESCRIPTION
**Pull request by codecaiine**
### Add contact form

- Add contact form
- Add form styles in main.css file

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+
+.form > button:hover {
+  background-color: #a04300;
+}
+
+.form > input {
+  border: none;
+  border-bottom: 1px solid grey;
+  outline: none;
+}
```
Comment (BertrandConxy) [inline css/main.css:364]: - [ ] Again, looking at the design of your form, I realized that it is almost similar to the Figma design given, but you should stick to the design as much as possible. In addition, as a Front-end Developer, attention to all details even the smallest ones is one of the most important abilities to have.
**Figma design**
image

Your Design
image
You can observe that there are some improvements that need to be made. Therefore, I would recommend you add them before proceeding to other tasks. You may consider increasing the size of the inputs, their `margin-bottom` and some `padding` to the placeholder and other things which are there in the design. 😺

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi  @codecaiine , 

**Good job so far! 🚀**

I really liked your design and your commitment to parse the Figma design. However, there are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃 


## Highlights 🏅 
- All linters are passing
- Good commit messages
- The project works fine
- Use of correct GitHub flow


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@BertrandConxy) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+  justify-content: center;
+}
+
+.primary-text {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  color: #262626;
+}
+
```
[the remark this answers is not in the document]
Comment (codecaiine) [reply css/main.css:381]: Thank you I'm implementing it.

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+  ```
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/codecaiine)
+
+
+## 🤝 Contributing
```
[the remark this answers is not in the document]
Comment (codecaiine) [reply README.md]: Thank you I'm implementing it.

File: README.md [hunk 4]
```diff
@@ -1 +1,47 @@
-# contact-form-review
+![](https://img.shields.io/badge/Portfolio-blue)
+# Portfolio
+> Portfolio is an app that allows you to showcase your different projects and provide a place where people can have chat with you.
+
+![screenshot]()
+
+
+[Live Demo]()
+
```
[the remark this answers is not in the document]
Comment (codecaiine) [reply README.md]: Thank you I'm implementing it.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+          </p>
+          <ul class="project-stacks card-stacks">
+            <li>HTML</li>
+            <li>SASS/CSS</li>
+            <li>JavaScript</li>
+            <li>WebPack</li>
+            <li>Babel</li>
+          </ul>
+          <a href="" class="card-link link" target="_blank"
+            rel="noreferrer noopener">See project</a>
```
Comment (mwanawabangona) [inline index.html:113]: - [ ] :wave: Kindly add a link to `href` which redirects to the source code. It is imperative that this is done so as to allow potential recruiters to check your Projects at their convenience.

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
# Status: Changes Required  
### Hi @codecaiine  :wave:
Wow 👨🏽‍💻 ! You've done an amazing job with this code review! You're rocking and only with minor :pinching_hand:  improvements which you left out, you're ready to launch to space and beyond 🚀 
> 
## Highlights  :medal_sports:

✅  Good utilization of GitHub inline comments feature in the review ✔️  
✅ All linters are passing
✅ Good commit messages
✅ Use of correct GitHub flow

## Required Changes 
Check the comments under the review.
##  Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**